packages/ui/src/components/Banner/__stories__/Image.tsx

```diff
@@ -0,0 +1,31 @@
QPVO8YaXmJRLrE+Q0SGRQgCrc9wiZ5GkpC6N5HcxIhkl/vmQBAi/rRvjcfDC72kE4Wlkj+Ku5T4ImE4wT6WPU2ASSKUvoq8xFIQCvkFlfZG4Q+FEIU/FMJqDH9IGuIvOBrD7psG8YG7w/HYzqSAT72ygldeX8RXn9+YUA2LnLRVTuQrmPryMy09LXWE42ZQVoiHuKWvXXheALnCnDjnHXQyrPL/duYK0z3XJINww8eFadsHaDiQh4byNbKr7hlWJ70s77DmxG6TIOMZ61oCjHM70kp6iRALPYKHfzLbJsLgJsFqi91jDMBssYDFcolPGkBgfwjss8mDbM+LpUla7jCKYhbh4tlryVHXLkMqo8mwQiIS60guwjoMdltbfeGQXrTfXvezs820MhJ455pyuWDesKbMchXGA4z8YQMGqi9dQyyaX2sg/dZx4Az+3Pjs3hVM/PsSUKBMFuh8UNA9/ul+iBGWIwueuQj83e44xlaAWz+hN0+YFR7xXdcn2vY93mzVPmokavQsXvY60dUDxJdW1POKvYxDip0A4PdLx/trTh4j08/22U6EgBCRYRGCACUmjYILVrNYQIWgKrEKHCWjo6OyVroSs/ecUG9WDxFfJGwneIZF+b8kbcraDaO9KlDoayghARHOedhrfe218IdQ8rBTuV7DD39IwmjOwMfvSOKuzwxg57EgB9w2vvb8Cl75gSJufFUZi0NhEGD0A3v+7UdbecJ4GigrVGH7JPpkOYm1FT4RqfMIFa9ah8GtKWLZ/MkCMvdyo3YrI5wzTh5+tFP3fFuQxu2hEGNNUqwxvLVCJzKuRWUSzNFELbdphiIt5BeNKOgjLBzLo7JQ6fAGs+AiwRweY/uzjNxhtP3IlaEIWd5hR+mhGZObk0CSduJyyBHROs32kYge0jZnPadNw7luuMtJ8h2L4WGhnM9Ee0Znn0nyMv121aGEPaSdC6xzxGsZqS9E3QIyzHIyUZZ+soi5w/Rze2jbYD1cog2z5w9g/K0TwAj/A+Xc7iWsv5vB2IgM6qLjXgCeWd0g99ZpA7f/XQVTT+llTj7xlgQObaDkQOSA945EC5NI3Orzh/yuin1EfCWrNQ8r72CTYvwQjHpwEYCHMDJ9XUD2IviIiAyLEAT2qNgQ9SZjjtkkjk2Q7OLeC2wC/ZIzTBqMGyj1Pq7ZjtbumMBQPB7l//LZpmrdKP+XSL1wzsVezv8lD4UdrKG9SgSYLMIgJyWGfCsU4u7PDOCd94Xjpd3MC/b+Pyjj6XVm4OEPeRW3/OQU1j+W6ahRX+mswzLtOvStOcrOgqkC+YttFfB2yXP+5Exq2Yn/SUs7EPRN2l/40Ztwvdh8AGke2Zpja5BgqA1xthxizXCJsc494Xk4JFTUcptmjB9K5BeRaFBpaG/h4Z/OEr3BaiB5jJnA07kVZCu2EGGihIFHT7LJp+jeYelEDEsbOfmVvJTJbBcNC+gss+8T2nnoPCft15ZIHS914azD6pdtg7OcRAJ52d80eD3mrXKGgKynl0dM3JvFYpYebm/iVWNYTNXDImZftQbrr+B/mJy3wi9+eR7paQrRpnuME9UlLCcoqN2ugMzhEi59SG+esPsujLnDIzIg+46UG/A+oSQz/+jEyEnOhBm7wPscW1AeXw3dzQ9FJgqxs7TodZNiGr7il8anMTL9jShsYm8jIsMi+I4b9u5cEBlllcZhUbLLi12TsEqp1C85wyw8dmLJvZFz19V6bw2Q+HIuFqbGPEzkhTFJoDA5EeX/EoRCX6P8XwIIK8RelP9LsG64+b9E0PQGu/3mAWyeD36WdimFWl6wN76nhJ+eI5kDQOl6Fa+85lgeZ31/piXaqiH0RbjBFbXDyhNmeYYFifyOFEYvI7+TZnMVrHx9Vrk1fpxZ4nNQ+q37c6UEf/21TXu3bbRygtX/2vOGWb9h1L3Dan81eYFpuVVTlGglv3oErePlWFSxcLwAc8XkeoMBnWVPLhO8w0T2qRBJ4diWr+LwDMM7zAoXl7CTPBQDzDIK+UGrQySGCGQQrdyu17md+JcTRpG1wFnmIMWcf53bSOvOfjplefuGu51BRrHOIS/nF0mGWa6JKLPKS8DYvexwiWNXjiH95nEMbR/kKATmjheQuuUUcJJCsAnflAUFxW/yIclp7Ie1Tw/rTbVhhUd881/znyf1hz/0kSRqoFoxOd5ebOKLC61fpHPgIL5M019zNbRJsT+xol37aYqC19WcPqKwiT2LiAyLEBREkh7KgcdMkcDxBpNRSfNYGxzkP4j1AnL5CvPm2U/ElxLCmDBQIgCi/F9C8NzeKP+XEMIKsRfl/xKo1335v1gI2xvsxldVcPGHC7jtBRJfwipdr94q28Mj8ubBSCvEU6E1V9hZeGFB7sMgHcPW9uvoXmGHv3oclUW9idwjqMOX25WHCcWFZLkjFKJFhKGx3txmtPKGybaH7gXmGYQhwDP5RSIIAgSNwPKy0A+Buu6nppc6vMHsJJhp+233FDuUXUapSvk4QjT8moTM5iP0DzEs77DcOse0D48wobWBR9q4SB6a95Phrksqd5JProVBglHrMJaOOgTdxDYR2kraf6L7h7SPWDp4dXhlrHaI2rbL8CArc7yE2d8wwhlOJIAR/rTm7CNZTNwWUH4wKTlBwbDaJ4KVKrC/qFFhHe+6PonFYXKZTgIsN6BS29QQ9si9pSu9vWy2AiW+WMhMfQOZKcvr7NoQSLGtjbCJ1wRsN4IGRGRYhKBAT9QgPMILVCEUmCQ3MaICjg0GkWYvGkz1Bxl27Jn2F42rlfjaRfIOC2vSQGlCPcr/xYVCX6P8X4III8RelP9LsG535v+i2zQwmg/XG+z+7VW88EP1vGBL7hSTbnjeT94PbPM8Ouc7x2ueYW7VnfoM1wqnWXATYQOmgc2lYJ+DSsMxnPH6SWp59m8PM+v7cfYEf0Z2D3qm742GLiRLrVCIrrCIzXCJMcnxUacXGGUI0EJ++QweydQrOPzEMoxi2xsMg
```

Comment:
You exported a png in a svg that's not the best way of doing and it breaks the structure of storybook